src/shared/database/amendments.ts

```diff
@@ -10,77 +14,163 @@ const log = logger({ name: 'amendments' })
 
 const amendmentIDs = new Map<string, { name: string; deprecated: boolean }>()
 const rippledVersions = new Map<string, string>()
-
-const ACTIVE_AMENDMENT_REGEX =
-  /^\s*REGISTER_F[A-Z]+\s*\((?<amendmentName>\S+),\s*.*$/u
-const RETIRED_AMENDMENT_REGEX =
-  /^ .*retireFeature\("(?<amendmentName>\S+)"\)[,;].*$/u
+// TODO: Use feature RPC instead when this issue is fixed and released:
+// https://github.com/XRPLF/rippled/issues/4730
+const RETIRED_AMENDMENTS = [
+  'MultiSign',
+  'TrustSetAuth',
+  'FeeEscalation',
+  'PayChan',
+  'CryptoConditions',
+  'TickSize',
+  'fix1368',
+  'Escrow',
+  'fix1373',
+  'EnforceInvariants',
+  'SortedDirectories',
+  'fix1201',
+  'fix1512',
+  'fix1523',
+  'fix1528',
+]
 
 const AMENDMENT_VERSION_REGEX =
   /\| \[(?<amendmentName>[a-zA-Z0-9_]+)\][^\n]+\| (?<version>v[0-9]*\.[0-9]*\.[0-9]*|TBD) *\|/u
 
-// TODO: Clean this up when this PR is merged:
-// https://github.com/XRPLF/rippled/pull/4781
+export const NETWORKS_HOSTS = new Map([
```

Comment:
Should this list come from the database.  That way more networks would be supported.

Reply:
nvm i just realized this moved.